README.md

```diff
@@ -1,23 +1,23 @@
-# React Enlightenment
+# React 깨우치기
 
-Written by [Cody Lindley](http://codylindley.com/) sponsored by — [Frontend Masters](https://frontendmasters.com/)
+[Frontend Masters](https://frontendmasters.com/)의 후원으로 [Cody Lindley](http://codylindley.com/)에 의해 작성되었습니다.
 
-Learn React in the terse cookbook style found with previous Enlightenment titles (i.e., [jQuery Enlightenment](http://jqueryenlightenment.com/), [JavaScript Enlightenment](http://javascriptenlightenment.com/), [DOM Enlightenment](http://domenlightenment.com/))
+React를 간결한 요리책처럼 배워보세요. 이런 방식은 이전의 깨우치기 출판물에서도 보실 수 있습니다. (예: [jQuery Enlightenment](http://jqueryenlightenment.com/), [JavaScript Enlightenment](http://javascriptenlightenment.com/), [DOM Enlightenment](http://domenlightenment.com/))
 
 ***
 
-**Read Online At**: 
+**온라인에서 읽기**: 
 
 * [reactenlightenment.com](http://www.reactenlightenment.com/)
 
-**Download a .pdf, .epub, or .mobi file from**:
+**.pdf, .epub, 또는 .mobi 파일로 다운로드하기**:
 
 * [https://www.gitbook.com/book/frontendmasters/react-enlightenment/details](https://www.gitbook.com/book/frontendmasters/react-enlightenment/details)
```

Comment:
As the original author refers to the Gitbook page, you can add our reference here: https://www.gitbook.com/book/hayanmind/react-enlightenment-in-korean/details